### PR TITLE
[Implementation] (Proposal 2)  Add Benchmark Pipeline 

### PR DESCRIPTION
**Pull request by dipankardas011**
#### What type of PR is this?
kind/feature

#### What this PR does / why we need it:
It will give us a alternative route to use kubernetes manifests for the benchmark jobs instead of reusing github workflow.

With this we are trying to use the manifest location in github or any public facing endpoint from where we can get them

#### Which issue(s) this PR fixes:

Addresses part of #121

#### Special notes for your reviewer (optional):

Created this for finding alternative route for the current proposal 2 impl

Refer: #124, https://github.com/cncf-tags/green-reviews-tooling/issues/121#issuecomment-2476378468

### CODE REVIEW

**Review by rossf7 — COMMENTED** [review]
@dipankardas011 Looks good just a few minor comments. We should also update proposal 2 since this changes the design.

File: .github/workflows/benchmark-pipeline.yaml [hunk 1]
```diff
@@ -76,11 +86,38 @@ jobs:
...
+          --all \
+          --for=condition=Ready
+
+      - name: Wait for the benchmark job to complete
+        run: |
+          sleep ${{ inputs.benchmark_job_duration }}m
+
+      # - name: Download the benchmark results
+      #   run: |
+      #     kubectl cp benchmark:/results results
```
[the remark this answers is not in the document]
Comment (dipankardas011) [reply .github/workflows/benchmark-pipeline.yaml:112]: Let's remove it

**Review by rossf7 — COMMENTED** [review]
@dipankardas011 Some more renaming is needed due to the param name change.

Please check if any more renaming is required and then I think this is good to go.

**Review by rossf7 — APPROVED** [review]
LGTM

I created a fix for CI issue in #130

File: projects/projects.json [hunk 2]
```diff
@@ -3,7 +3,10 @@
         {
             "name": "falco",
             "organization": "falcosecurity",
-            "benchmark_path": "",
+            "benchmark": {
+                "k8s_manifest_url": "https://raw.githubusercontent.com/falcosecurity/cncf-green-review-testing/refs/heads/main/benchmark-tests/falco-benchmark-tests.yaml",
```
Comment (dipankardas011) [inline projects/projects.json:7]: @rossf7 Does it makes sense?

```
https://raw.githubusercontent.com/falcosecurity/cncf-green-review-testing/e93136094735c1a52cbbef3d7e362839f26f4944/benchmark-tests/falco-benchmark-tests.yaml
```

Comment (rossf7) [reply projects/projects.json:7]: @dipankardas011 Yes, looks good. Thx!

**Comment by nikimanoledaki** [comment]
@dipankardas011 could you rebase with `main` to get CI to pass please?  :)

**Review by nikimanoledaki — APPROVED** [review]

**Pull request merged**